src/app/containers/SocialEmbed/helper.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { ToggleContext } from '#contexts/ToggleContext';
+
+/**
+ * withContexts is a higher-order component that returns a React component
+ * wrapped in the necessary page contexts for testing and Stories.
```

Comment:
maybe this file should be called `test-helper.jsx` to make clear it is not part of the social embed implementation?

Reply:
Good idea – I've renamed it.